quiche/src/recovery/mod.rs

```diff
@@ -555,11 +683,7 @@ impl Recovery {
             let latest_rtt =
                 now.saturating_duration_since(largest_newly_acked_sent_time);
 
-            let ack_delay = if epoch == packet::Epoch::Application {
-                Duration::from_micros(ack_delay)
-            } else {
-                Duration::from_micros(0)
-            };
+            let ack_delay = Duration::from_micros(ack_delay);
```

Comment:
This also seems like a functional change since it will affect how `update_rtt` works, albeit related to https://github.com/cloudflare/quiche/pull/1742/commits/a79dbfa65401f5e6497b2984988ea5572fb33bce#r1555902791